assume/common/market_objects.py

```diff
@@ -136,6 +138,8 @@ class MarketConfig:
     price_tick: float | None = None  # steps in which the price can be increased
     supports_get_unmatched: bool = False
     eligible_obligations_lambda: eligible_lambda = lambda x: True
+    network_path: str = " "
+    solver: str = "glpk"
```

Comment:
I don't like having these here a lot - this does not have a lot todo with how the market is configured concretely - I would suggest to have something like the bidding_params with
`bidding_params = unit_params.get("bidding_params", self.bidding_params)`

so instead of these two add a dict with `market_params` which contains these

Reply:
please make a suggestion in code